docs/pages/icons/IconTable.vue

```diff
@@ -132,6 +132,15 @@
         }
         return false;
       },
+      blackIconFilter(alias) {
+        return !KolibriIcons[alias].defaultColor && !KolibriIcons[alias].learningActivity;
+      },
+      iconWithDefaultColorFilter(alias) {
+        return Boolean(KolibriIcons[alias].defaultColor);
+      },
+      learningActivityIconFilter(alias) {
+        return KolibriIcons[alias].learningActivity;
+      },
```

Comment:
Minor stylistic note:

Since all of these are pure functions taking `alias` without reference to `this`, they can actually be made normal helper functions at the module level rather than making them component methods. This distinction can be helpful for future refactors and understanding the component API.